gulpfile.js

```diff
@@ -0,0 +1,26 @@
+'use strict'; //eslint-disable-line
+
+const gulp = require('gulp');
+const fs = require('fs');
+
+// Read svg directory for list of logo names to pass to demo template
+const logoNames = fs.readdirSync('src').filter((file) => {
+
+	// File is an SVG
+	return file.indexOf(".svg") > -1;
```

Comment:
Ah and that would be a more natural way of reading this if statement and therefore better because it would be easier for a human to understand what the code is doing quicker?